src/twisted/python/_release.py

```diff
@@ -502,12 +503,26 @@ def main(self, args):
         @param args: The command line arguments to process.  This must contain
             two strings: the path to the root of the Twisted checkout, and a
             path to an output directory.
+
+        @return: The list of build errors.
+        @rtype: List of L{str}
+
         """
         if len(args) != 2:
             sys.exit(
                 "Must specify two arguments: " "Twisted checkout and destination path"
             )
-        self.buildAPIDocs(FilePath(args[0]), FilePath(args[1]))
+
+        original_stdout = sys.stdout
+        try:
+            capture = StringIO()
```

Comment:
https://docs.python.org/3/library/contextlib.html#contextlib.redirect_stdout

Reply:
True.. I will use it. I am not familiar with latest stdlib :) Nice to have this helper in stdlib.